D1371R3.md

```diff
@@ -385,29 +384,27 @@ Within the parentheses, `inspect` is equivalent to `switch` and `if` statements
 except that no conversion nor promotion takes place in evaluating the value
 of its condition.
 
-There are two clause variations types inside the `inspect` body:
-
-  - `=>` denotes expression yielding a value.
-  - `:` never yields a value, but instead denotes a statement that is executed
-    in the enclosing context. e.g, a `return` statement will return from the
-    function `inspect` is currently running in.
```

Comment:
I think it would be valuable to have this in some sort of "alternatives" section. Maybe somewhere under "Design Decisions"? The idea is that we have a fallback proposal we can vote on, in case the "always" expression approach doesn't fly.